parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -648,7 +689,8 @@ public void writeDataPageHeader(
       org.apache.parquet.column.Encoding dlEncoding,
       org.apache.parquet.column.Encoding valuesEncoding,
       OutputStream to) throws IOException {
-    writePageHeader(newDataPageHeader(uncompressedSize,
+    writePageHeader(newDataPageHeader(createdBy,
```

Comment:
It looks like these changes apply to the write side, but we need to ignore statistics when reading.